core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -1554,9 +1570,11 @@ class UnifiedLog(@volatile var logStartOffset: Long,
   }
 
   /**
-   * Completely delete the local log directory and all contents from the file system with no delay
+   * Completely delete the local log directory and all contents from the file system with no delay.
+   *
+   * Visible for testing.
    */
-  private[log] def delete(): Unit = {
+  def delete(): Unit = {
```

Comment:
nit: Could we keep it package private? It does not seem to be used by other packages.

Reply:
Fixed.